app/components/network/network.html

```diff
@@ -65,3 +65,34 @@
     </rd-widget>
   </div>
 </div>
+
+
+<div class="row" ng-if="!(network.Containers | emptyobject)">
+  <div class="col-lg-12 col-md-12 col-xs-12">
+    <rd-widget>
+      <rd-widget-header icon="fa-server" title="Containers in network"></rd-widget-header>
+      <rd-widget-body classes="no-padding">
+        <table class="table">
+          <thead>
+            <th>Container Name</th>
+            <th>IPv4 Address</th>
+            <th>IPv6 Address</th>
+            <th>MacAddress</th>
+            <th>Actions</th>
+          </thead>
+          <tbody>
+            <tr ng-repeat="container in containersInNetwork">
+              <td><a ui-sref="container({id: container.Id})">{{ container.Name }}</a></td>
+              <td>{{ container.IPv4Address || '-' }}</td>
+              <td>{{ container.IPv6Address || '-' }}</td>
+              <td>{{ container.MacAddress || '-' }}</td>
+              <td>
+                <button type="button" class="btn btn-xs btn-danger" ng-click="containerLeaveNetwork(network, container.Id)"><i class="fa fa-trash space-right" aria-hidden="true"></i>Leave Network</button>
```

Comment:
Same here, the containerLeaveNetwork action won't work as there is no `Id` field in the struct.